pinot-query-planner/src/main/java/org/apache/pinot/query/parser/CalciteRexExpressionParser.java

```diff
@@ -50,6 +52,8 @@
 public class CalciteRexExpressionParser {
   private static final Logger LOGGER = LoggerFactory.getLogger(CalciteRexExpressionParser.class);
   private static final Map<String, String> CANONICAL_NAME_TO_SPECIAL_KEY_MAP;
+  private static final Set<String> ARRAY_TO_MV_ALIAS =
```

Comment:
Do we need this many variance? Once we have unnest this should be removed right?

Reply:
done